### PR TITLE
Update uperf execution aligned with pull/324

### PR DESCRIPTION
**Pull request by morenod**
Required changes on airflow aligned with https://github.com/cloud-bulldozer/e2e-benchmarking/pull/324

### CODE REVIEW

**Comment by Harshith-umesh** [comment]
@morenod could you also update this file: https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/config/baremetal-benchmarks/scaleup-bench.json

**Comment by morenod** [comment]
> @morenod could you also update this file: https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/config/baremetal-benchmarks/scaleup-bench.json

Done

File: dags/openshift_nightlies/config/benchmarks/openstack.json [hunk 1]
```diff
@@ -80,34 +80,32 @@
...
-            "command": "./run_hostnetwork_network_test_fromgit.sh test_cloud",
+            "command": "./run.sh",
             "env": {
-                "COMPARE": "false",
-                "ENABLE_SNAPPY_BACKUP": "true",
-                "MULTI_AZ": "false",
-                "EMAIL_ID_FOR_RESULTS_SHEET": "mkaliyam@redhat.com"
+                "WORKLOAD": "hostnet",
+		"MULTI_AZ": "false",
+		"COMPARE": "false",
```
Comment (rsevilla87) [inline dags/openshift_nightlies/config/benchmarks/openstack.json:87]: Fix linting

**Review by rsevilla87 — APPROVED** [review]
LGTM; just a minor nit

File: dags/openshift_nightlies/config/benchmarks/openstack.json (hunk 1 above)
Comment (morenod) [reply dags/openshift_nightlies/config/benchmarks/openstack.json]: done

File: dags/openshift_nightlies/config/benchmarks/openstack.json [hunk 2]
```diff
@@ -80,34 +80,32 @@
         {
             "name": "host-network",
             "workload": "network-perf",
-            "command": "./run_hostnetwork_network_test_fromgit.sh test_cloud",
+            "command": "./run.sh",
             "env": {
-                "COMPARE": "false",
-                "ENABLE_SNAPPY_BACKUP": "true",
+                "WORKLOAD": "hostnet",
```
Comment (rsevilla87) [inline dags/openshift_nightlies/config/benchmarks/openstack.json:85]: The workloads are not configured properly

According to the docs:
```
1. pod to pod using SDN: `WORKLOAD=pod2pod ./run.sh`
2. pod to pod using SDN and network policy: `WORKLOAD=pod2pod NETWORK_POLICY=true ./run.sh`
3. pod to pod using Hostnetwork: `WORKLOAD=pod2pod HOSTNETWORK=true ./run.sh`
4. pod to service: `WORKLOAD=pod2svc ./run.sh`
5. ```

**Review by rsevilla87 — CHANGES_REQUESTED** [review]

File: dags/openshift_nightlies/config/benchmarks/openstack.json (hunk 2 above)
Comment (morenod) [reply dags/openshift_nightlies/config/benchmarks/openstack.json:85]: done

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Pull request merged**